node/node.go

```diff
@@ -374,76 +337,71 @@ func (node *Node) Start() error {
 			return err
 		}
 		topicNamed := allTopics[i].Name
+
+		if nodeconfig.GroupID(topicNamed) == nodeconfig.NewGroupIDByShardID(
+			nodeconfig.ShardID(node.Consensus.ShardID),
+		) {
+
+			pubsub.RegisterTopicValidator(
+				topicNamed,
+				func(ctx context.Context, peer libp2p_peer.ID, msg *libp2p_pubsub.Message) bool {
+					sig2 := msg.GetData()
+					if len(sig2) < p2pMsgPrefixSize {
+						return true
```

Comment:
return false here?

Reply:
No because that will invalidate the message . There are messages that are still valid to go through here , basically beacon based things